parallel_algorithms/P3179/P3179.bs

```diff
@@ -821,24 +853,34 @@ operations <a href="https://eel.is/c++draft/numeric.ops">[numeric.ops]</a>. The
 adding overloads with `ExecutionPolicy`  to the existing algorithms in the `std::ranges` namespace. Any follow-up paper
 that adds `<numeric>` algorithms to `std::ranges` should also consider adding dedicated overloads with `ExecutionPolicy`.
 
-### The rest of `std::ranges` only algorithms ### {#rest_std_ranges_algorithms}
+# Implementation experience # {#impl_exp}
 
-// TODO: Add a full list
+The oneAPI DPC++ Library (oneDPL) [developer guide](https://oneapi-src.github.io/oneDPL/parallel_api/parallel_range_algorithms.html)
+covers parallel range algorithms we’ve implemented so far. The oneAPI specification provides
+[formal signatures](https://github.com/uxlfoundation/oneAPI-spec/blob/main/source/elements/oneDPL/source/parallel_api/parallel_range_api.rst)
+of these algorithms. The implementation supports execution policies for CPUs (semantically aligned with the C++ standard)
+and for SYCL devices, and it works with a subset of the standard C++ views.
+
+We use the range-as-the-output approach where applicable: in `transform`, `copy`, `copy_if`, and `merge`.
+
+We don't forsee any issues with implentability for the rest of the proposed parallel algorithms because all of them were
+already implemented in C++17 and new APIs that we propose are expressable via the existing ones.
 
 # Further work # {#further_work}
 
 ## Issues to address ## {#issues_to_address}
 
-Based on the feedback from SG1 and SG9, the plan of work prior to the next committee F2F meeting in 2025 is:
-- <s>Fix bugs in the signatures.</s>
-- Update the synopsis of <algorithm>.
-- Add the algorithm descriptions to the wording, even if no changes are made.
-- Add the necessary poll results.
-- Add implementation experience (copy from 3490) and thoughts on implementability.
-- Decide on constraining the execution policy template parameter.
-- Add "The intention is that algorithms call begin/end only once, in serial code" to the design notes.
-- List all the algorithms that are being given execution policy overloads with a brief justification.
-- List range algorithms that are being excluded, with a brief justification.
+Having known bugs and feedback from SG1 and SG9, the plan of work prior to the next committee F2F meeting in
+Austia, 2025 is the following:
+- <s>Fix known bugs in the signatures wording.</s>
+- <s>Decide on constraining the execution policy template parameter.</s>
+- <s>List all the algorithms that are being given execution policy overloads.</s>
+- <s>List range algorithms that are being excluded.</s>
+- <s>Add implementation experience and thoughts on implementability.</s>
```

Comment:
Why not just add these to the R3 => R4 list, instead of striking through?

Reply:
Move to revision history, split on bullets and sub-bullets (to address the SG1 and SG9 feedback)